dm/checker/checker.go

```diff
@@ -550,16 +547,6 @@ func (c *Checker) Init(ctx context.Context) (err error) {
 	return nil
 }
 
-func lightningCheckGroupOnlyTableEmpty(checkingItems map[string]string) bool {
```

Comment:
why remove this check

Reply:
To fix this issue https://github.com/pingcap/tiflow/issues/11945. Root cause is this check will set pd addr to `noop:2379`. In fact, when we run checker by `RunCheckOnConfigs()` https://github.com/pingcap/tiflow/blob/master/dm/checker/cmd.go#L113, it will initial all checker and remove few ignored checkers https://github.com/pingcap/tiflow/blob/master/dm/checker/cmd.go#L49. But the rest checker do not ignored, will use pdClient to get some info.